ndcube/mixins/plotting.py

```diff
@@ -298,7 +298,10 @@ def _plot_3D_cube(self, plot_axis_indices=None, axes_coordinates=None,
         plot_axis_indices = [i if i >= 0 else self.data.ndim + i for i in plot_axis_indices]
         # If axes kwargs not set by user, set them as list of Nones for
         # each axis for consistent behaviour.
-        if axes_coordinates is None:
+        # Or if the axes_coordinates values are None for plot_axis_indices indexes
+        # This ensures no extraneous values from the user.
+        if axes_coordinates is None or  (axes_coordinates[plot_axis_indices[0]] is None and
+                axes_coordinates[plot_axis_indices[1]] is None):
```

Comment:
Why this extra condition in the if statement? What if the plot axes coordinates are ```None```, but the user has entered coordinates for one or more slider axes? That should be supported.

Reply:
Sure, this seems to be a logical error on my behalf, I would correct it.